docs/migrate-from-sidecar-to-ambient/migration.adoc

```diff
@@ -0,0 +1,2114 @@
+** <<step-7-enable-waypoint-processing, Step 7: Enable Waypoint Processing>>
+*** <<step-71-activate-waypoints-for-l7-processing, Step 7.1 Activate Waypoints for L7 Processing>>
+*** <<step-72-validate-l7-policy-enforcement, Step 7.2 Validate L7 Policy Enforcement>>
+** <<step-8-policy-simplification, Step 8: Policy Simplification>>
+*** <<step-81-remove-redundant-sidecar-policies, Step 8.1 Remove Redundant Sidecar Policies>>
+** <<step-9-remove-sidecar-injection, Step 9: Remove Sidecar Injection>>
+*** <<step-91-disable-sidecar-injection, Step 9.1 Disable Sidecar Injection>>
+*** <<step-92-validate-sidecar-removal, Step 9.2 Validate Sidecar Removal>>
+* <<validation-and-testing, 4. Validation and Testing>>
+** <<migration-verification, 4.1 Migration Verification>>
+* <<troubleshooting, 5. Troubleshooting>>
+** <<common-issues, 5.1 Common Issues>>
+** <<rollback-procedures, 5.2 Rollback Procedures>>
+* <<post-migration-optimization, 6. Post-Migration Optimization>>
+** <<waypoint-right-sizing, 6.1 Waypoint Right-Sizing>>
+** <<observability-considerations, 6.2 Observability Considerations>>
+** <<configure-kiali-for-ambient-mode, 6.3 Configure Kiali for Ambient Mode>>
+* <<additional-resources, 7. Additional Resources>>
+
+---
+
+[[overview-of-sidecar-to-ambient-migration]]
+== 1. Overview of Sidecar to Ambient Migration
+
+[[migration-strategy]]
+=== 1.1 Migration Strategy
+
+The migration from sidecar to ambient mode follows a structured, phased approach to ensure a smooth transition with minimal disruption. Key principles include:
+
+1. **Gradual Transition**: Both sidecar and ambient proxies operate simultaneously during migration
+2. **Service-by-Service**: Migration can occur incrementally, one service or namespace at a time
+3. **Policy Preservation**: All existing security and traffic policies are maintained throughout the process
+4. **Rollback Support**: Each step can be reversed if issues arise
+
+[[key-architectural-changes]]
+=== 1.2 Key Architectural Changes
+
+[cols="1,2,2"]
+|===
+|Component |Sidecar Mode |Ambient Mode
+
+|**L4 Processing**
+|Envoy sidecar per pod
+|ZTunnel node proxy (shared)
+
+|**L7 Processing**
+|Envoy sidecar per pod
+|Waypoint proxy per namespace/service
+
+|**mTLS Enforcement**
+|Sidecar proxy
+|ZTunnel proxy
+
+|**Policy Enforcement**
+|Sidecar proxy
+|ZTunnel (L4) + Waypoint (L7)
+|===
+
+---
+
+[[prerequisites]]
+== 2. Prerequisites
+
+For this migration we are going to use the bookinfo application as an example. You can use any application that is currently running with sidecar injection enabled.
+
+[[technical-requirements]]
+=== 2.1 Technical Requirements
+
+Before starting migration, ensure your environment meets these requirements:
+
+For Kubernetes:
+
+- **Kubernetes 1.21+**: Minimum version for ambient mode support
+- **Sail Operator v1.27+**: Must be running Sail Operator version 1.27 or later
+- **Istio v1.27+**: Control plane must be v1.27 or later for ambient support
```

Comment:
No action required: while this is true for downstream, the previous versions include ambient support as well.